invenio_rdm_records/templates/semantic-ui/invenio_notifications/community-submission.submitted.jinja

```diff
@@ -0,0 +1,24 @@
+{# notifications/community-submission.submitted.jinja #}
+
+{%- block subject -%}
+New record submission for your community {{ notification.context.get("request").get("receiver").get("metadata").get("title") }} submitted by {{ notification.context.get("request").get("created_by").get("username") }}
+{%- endblock subject -%}
```

Comment:
```suggestion
{# notifications/community-submission.submitted.jinja #}

{% set community = notification.request.receiver %}
{% set creator = notification.request.created_by %}

{%- block subject -%}
New record submission for your community {{ community.metadata.title }} submitted by {{ creator.username }}
{%- endblock subject -%}
```

minor: would something like this work? Since we're in Jinja, we should take advantage such tricks as much as we can, otherwise templating will become very tedious/verbose :)

Reply:
We could use the `dict_lookup` method (if provided for jinja templates) or at least remove the `.get(...)'  calls and just access them via ["..."].

Also taking into account setting variables as suggested by you.

```suggestion
{# notifications/community-submission.submitted.jinja #}

{% set community = notification["context"]["request"]["receiver"] %}
{% set creator = dict_lookup(notification, "request.created_by") %}

{%- block subject -%}
New record submission for your community {{ community["metadata"]["title"] }} submitted by {{ dict_lookup(creator, "username") }}
{%- endblock subject -%}
```

Not sure which one is better though :sweat_smile:

Reply:
Update: since we render blocks individually, they do not have access to variables set in another scope (i.e. outside of the block). With the change from notifications [here](https://github.com/inveniosoftware/invenio-notifications/pull/11), an initial block can take care of setting everything the template needs without touching `notifications` or `recipients`